web/src/lib/utils/helpers.ts

```diff
@@ -58,6 +59,19 @@ export const useScrollToPathAnchor = (): void => {
   });
 };
 
+export const getTaskStatusUpdatedMessage = (taskStatus: TaskProgress): string => {
+  const Config = getMergedConfig();
+
+  switch (taskStatus) {
+    case "NOT_STARTED":
+      return `${Config.taskDefaults.taskProgressSnackbarPrefix} "${Config.taskProgress.NOT_STARTED}"`;
```

Comment:
[dust] can we pull out the prefix before the switch? Just so it's clear they are all using the same prefix